hsm/sdf/SDFCryptoProvider.cpp

```diff
@@ -289,9 +295,9 @@ unsigned int SDFCryptoProvider::Verify(Key const& key, AlgorithmType algorithm,
         else
         {
             ECCrefPublicKey eccKey;
-            eccKey.bits = 32 * 8;
-            memcpy(eccKey.x, key.publicKey()->data(), 32);
-            memcpy(eccKey.y, key.publicKey()->data() + 32, 32);
+            eccKey.bits = 256;
+            memcpy(eccKey.x+32, key.publicKey()->data(), 32);
```

Comment:
make sure the 0-31 of eccKey.x/y is all set to 0

Reply:
are all 0